atlas/AtlasMl/atlasml/app.py

```
@@ -1,10 +1,13 @@
 import logging
 import os
+import json
+import time
 from contextlib import asynccontextmanager
 
-from apscheduler.schedulers.background import BackgroundScheduler
-from apscheduler.triggers.interval import IntervalTrigger
-from fastapi import FastAPI
+from fastapi import FastAPI, Request, Response, HTTPException
```

Ruff (0.12.2)

`fastapi.Response` imported but unused

Remove unused import

(F401)